### DISCUSSION TITLE
2024/04/me/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#104] [General]
# 2024/04/me/

https://xiangyun.rbind.io/2024/04/me/

**Comment by earfanfan** [comment]
湘云呐，你这是先下结论后展开写内容。不过我看完了以后，从我的视角看，你原来的结论也可以换个角度。

其一，你描述的自私，我觉得是划定自己和他人的边界，这样是很好的啊，有了边界就不会轻易地让渡自己的权益给别人了，更关注自己的内心，不就是在更多地发现自我么，人的一生都是在不断地认识自我的。

其二，你认识到自己的局限和变化，这也很好啊，大家都是这样的，都不是一出生就是完人，就像前些天我租车开回去，我妈担心，我就说会开车的人都不是一出生就会的。

其三，你描述的优柔寡断，额，其实也还好，谁说做决定一定要很快很快才好的。话说我清明去上海看那些展，回头想想好像就是接受了你的安利，以前你说这个好看那个好看，然后我就去看了。话说我决定养阿木狗还前前后后考虑了大半年才下定决心呢。

至于白头发嘛，我从小就是少年白，经历与你相似，现在也是不在意了，就是小时候总被当做异类，长大了自然就看开了。

老和新只是时间诞生上的早晚，你喜欢就喜欢呗，近几年流行国潮复苏，估计你出去溜达会碰到和你有共同兴趣爱好的人的。只不过，与你同行之人并不会无端从天而降，按照我的历史经验，是需要先打开自己的，不然的话，别人发出来什么到了我这里就会碰到钢铁一样的保护圈而被弹走，感觉比较好的状态应该是保护圈是隐形的。

哈哈，我以前看过网上一种说法，大意是说中国青年的30岁，才是别人的18岁。现在你还很年轻，就已经认识自己这么多这么清楚，真得很棒啊。话说我堂妹依依有次去看荷花，她鼓起勇气搭讪了一个超会摄影的美女姐姐，然后就知道了武汉看日落超美的地方。我上次去上海看展也是鼓起勇气和别人交流，也有些收获。下回你出去溜达的时候，你愿意的话，也可以和不同的人试着交谈，说不定会有意想不到的收获。

**Reply by XiangyunHuang** [reply]
央美毕业展吸引 40万人 来看，你敢信！我震惊了，靳尚谊在抖音上传的一段话成了效果最大的广告。其实，前两个月，陆续还去清华大学美术学院和中国工艺美术学院的毕业生作品展，央美的观众最多，有的画确实水平高。

绝大多数游客都是去拍照打卡的，央美美术馆人山人海，交流谈不上了，偶遇几位毕业生在画作前介绍，然后签名，我只是要了几张画作的明信片和签名，人家也很忙。更多的是作者不在现场，或者我没遇上。

**Reply by XiangyunHuang** [reply]
我写这段文字的时候，有真诚、有自嘲，还有得瑟哈，更多是后者。

**Reply by earfanfan** [reply]
我敢信呐，虽然我不看抖音没有听说过你提到的这个人类。另外，你的无形安利我又无形接受了，明年6月也找时间去这些学校看看毕业生作品展，哈哈，顺带看看你吧。

**Comment by earfanfan** [comment]
测试两下，这个样本是之前有过留言的。
